### PR TITLE
@skylar/swap-sheet-flow

### PR DESCRIPTION
**Pull request by skylarbarrera**
- Full Swap Sheet Flow
- Some e2e helper method changes; less flakey
- Updated Detox to latest version

### CODE REVIEW

File: ios/Extensions.swift [hunk 1]
```diff
@@ -210,7 +210,8 @@ extension UITableView {
         messageLabel.textAlignment = .center
         messageLabel.textColor = UIColor.RainbowTheme.Transactions.blueGreyDark35
         messageLabel.sizeToFit()
-        messageLabel.accessibilityIdentifier = "no-transactions-yet-label";
+        messageLabel.isAccessibilityElement = true
```
Comment (brunobar79) [inline ios/Extensions.swift:213]: You only need this if you're planning to tap on it

File: ios/Rainbow.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -754,6 +755,7 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEVELOPMENT_TEAM = L74NQAQB8H;
 				ENABLE_BITCODE = NO;
+				"EXCLUDED_ARCHS[sdk=*]" = arm64;
```
Comment (brunobar79) [inline ios/Rainbow.xcodeproj/project.pbxproj:758]: Can you try removing this line and running the tests in release mode? I don't want to make this change for the release builds.

File: ios/Rainbow.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 3]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-	<key>PreviewsEnabled</key>
-	<false/>
-</dict>
-</plist>
```
Comment (brunobar79) [inline ios/Rainbow.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:8]: Can you explain why is this file being removed?

File: ios/Extensions.swift (hunk 1 above)
Comment (skylarbarrera) [reply ios/Extensions.swift:213]: test fails without on iOS 14, can be verified on CI with latest tests

File: ios/Rainbow.xcodeproj/project.pbxproj (hunk 2 above)
Comment (skylarbarrera) [reply ios/Rainbow.xcodeproj/project.pbxproj]: switched to Simulator only

**Pull request closed without merging**